### PR TITLE
Wobble button

### PR DESCRIPTION
**Pull request by ShaikhHeeba07**

### CODE REVIEW

**Comment by Spyware007** [comment]
@ShaikhHeeba07 Align the button in centre

**Comment by ShaikhHeeba07** [comment]
Button is already in the center. Can you help me out how I can align the button in the center when its put in the actual website.
WhatsApp Image 2022-10-23 at 05 05 50

**Comment by ShaikhHeeba07** [comment]
@Spyware007 Please help me out

File: public/Buttons/ShaikhHeeba07/style.css [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+.btn-62 *,
+.btn-62 :after,
+.btn-62 :before,
+.btn-62:after,
+.btn-62:before {
+  border: 0 solid;
+  box-sizing: border-box;
+}
+body{
+   background-color: #CCFFBD;
```
Comment (Spyware007) [inline public/Buttons/ShaikhHeeba07/style.css:11]: add this here.
display:flex;
justify-content:center;
align-items:center;

File: public/Buttons/ShaikhHeeba07/style.css [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+}
+.btn-62{
+ 
+    position: absolute;
```
Comment (Spyware007) [inline public/Buttons/ShaikhHeeba07/style.css:15]: remove position absolute and respective things from here.

File: public/Buttons/ShaikhHeeba07/style.css (hunk 1 above)
Comment (ShaikhHeeba07) [reply public/Buttons/ShaikhHeeba07/style.css:11]: Do I have to add this in the body ?

**Comment by ShaikhHeeba07** [comment]
@Spyware007  Please do have a look .. I have commited the changes

**Comment by Spyware007** [comment]
@ShaikhHeeba07 Thanks for your contribution. 🤩
Leave a Star if you like it. 🌟

**Pull request merged**